services/QuillLMS/app/workers/save_user_pack_sequence_items_worker.rb

```diff
@@ -4,7 +4,9 @@ class SaveUserPackSequenceItemsWorker
   include Sidekiq::Worker
 
   sidekiq_options queue: SidekiqQueue::DEFAULT,
-    lock: :until_executed
+    lock: :until_executed,
+    on_conflict: :log,
+    lock_ttl: 2.minutes
```

Comment:
What is the reason for the lock_ttl? Is it a guard against locks that aren't cleaned up properly (for some unknown reason)?

If I'm reading the docs correctly, this `lock_ttl: 2.minutes` means that if a `SaveUserPackSequenceItemsWorker` takes 3 minutes to perform, if another `SaveUserPackSequenceItemsWorker` with the same params hits the queue at 2 minutes into processing, it will start processing as well, so two of them will be running at the same time. 

Looking at our transaction logs, we have had jobs take over 2 minutes. If this is solely a protection against uncleaned-up locks, we might want to make the `lock_ttl` longer. (Or remove it).

![Screenshot 2023-01-31 at 4 25 06 PM](https://user-images.githubusercontent.com/1304933/215887115-db556696-55b4-40d9-ab2d-9d33d211cbbe.png)

Reply:
I'm going to remove the `lock_ttl` for now.  I need to look into possibly configuring the reaper.